src/main/scala/org/ergoplatform/nodeView/mempool/ErgoMemPool.scala

```diff
@@ -72,6 +67,8 @@ class ErgoMemPool private[mempool](pool: OrderedTxPool)(implicit settings: ErgoS
             new ErgoMemPool(pool.invalidate(tx)) -> ProcessingOutcome.Invalidated(_),
             _ => new ErgoMemPool(pool.put(tx)) -> ProcessingOutcome.Accepted
           )
+        // transaction validation currently works only for UtxoState, so this branch currently
+        // will not be triggered probably
```

Comment:
This comment belongs to which branch?

Reply:
The branch below , added line break.

Reply:
I think it is better to put comment IN the scope they belong to.

Reply:
ok, done